src/app/meds-n-labels/activity-1/matchmeds.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="header">
+	<app-header (togglenavbar)="nav.toggle()"> </app-header>
+</div>
+<div class="container-fluid">
+	<app-navbar #nav (toggleposition)="toggle();ind.toggle()" (infopop)="info.pop()"> </app-navbar>
+  <app-infokit #info> </app-infokit>
+	<div class="row">
+		<div [className]="position">
+      <div class="meds">
+        <div >
+        <img src="assets/img/Match-meds/mefloquine.PNG" class="medicine" >
+        <p class="text-center"> Mefloquine </p>
```

Comment:
Remove these hard coded strings and add them in the property file. Check all hard coded strings at every HTML page and add them to property file